client/onboarding/utils.ts

```diff
@@ -69,48 +63,6 @@ export const finalizeOnboarding = async ( urlSource: string ) => {
 	} );
 };
 
-/**
- * Make an API request to determine if the user is eligible for a PO account.
- *
- * @param onboardingFields The form data, used to determine eligibility.
- */
-export const isPoEligible = async (
-	onboardingFields: OnboardingFields
-): Promise< boolean > => {
-	// Check if any required property is undefined
-	if (
-		hasUndefinedValues( {
-			country: onboardingFields.country,
-			business_type: onboardingFields.business_type,
-			mcc: onboardingFields.mcc,
-			annual_revenue: onboardingFields.annual_revenue,
-			go_live_timeframe: onboardingFields.go_live_timeframe,
-		} )
-	) {
-		return false;
-	}
-
-	const eligibilityData: PoEligibleData = {
-		business: {
-			country: onboardingFields.country as string,
-			type: onboardingFields.business_type as string,
-			mcc: onboardingFields.mcc as string,
-		},
-		store: {
-			annual_revenue: onboardingFields.annual_revenue as string,
-			go_live_timeframe: onboardingFields.go_live_timeframe as string,
-		},
-	};
-
-	const response: PoEligibleResponse = await apiFetch( {
-		path: `${ NAMESPACE }/onboarding/router/po_eligible`,
```

Comment:
Can we also remove the server logic that handles this request [here](https://github.com/Automattic/woocommerce-payments/blob/4edc665ee7bdba43ab1b66a9a10f83f03184d986/includes/admin/class-wc-rest-payments-onboarding-controller.php#L362), [here](https://github.com/Automattic/woocommerce-payments/blob/80c1a8578469702911ec7c2e7c51b424af3b1e7c/includes/wc-payment-api/class-wc-payments-api-client.php#L2135) and [here](https://github.com/Automattic/woocommerce-payments/blob/4edc665ee7bdba43ab1b66a9a10f83f03184d986/includes/admin/class-wc-rest-payments-onboarding-controller.php#L165) and any associated tests?

I did a quick check of the JS code and I can't see any requests being made to those endpoints after those changes, but wanted to double check with you first. 🙂